src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java

```diff
@@ -612,6 +616,71 @@ public void testBaseQualityFilter() throws IOException {
         Assert.assertFalse(vc.get().getFilters().contains(GATKVCFConstants.MEDIAN_BASE_QUALITY_FILTER_NAME));
     }
 
+    @Test
+    public void testReadBasedAnnotations() throws IOException {
+        // Test case 1: with the read correction we lose the variant - blood biopsy-like case
+        // Start creating bam file
+        final int numAltPairs = 5;
+        final int depth = 100;
+        final int refDepth = depth - 2*numAltPairs;
+        final byte altQuality = 50;
+        final byte refQuality = 30;
+
+        final File samFile = File.createTempFile("liquid-biopsy", ".bam");
```

Comment:
This setting-up could go in a `@BeforeClass` method.

Reply:
I tried to use `@BeforeClass` and ran into the issue where the `@BeforeClass` method takes in parameters (`numAltPairs` and `refDepth`) that I later use in `Assert`. Hardcoding them in the `@BeforeClass` method wouldn't be ideal. I could make them class-wide variables, since there are only two, but widening the scope of variables like that isn't ideal, either. So far I just extracted the sam file creation method without adding the `@BeforeClass` tag (in my latest commit). We can talk more tomorrow.